src/Home.jsx

```diff
@@ -1,8 +1,10 @@
 import React from "react";
-import { Button, Center, Heading, Link, Spinner, Text } from "@chakra-ui/react";
+import { Box, Button, Center, Heading, Link, Spinner, Text } from "@chakra-ui/react";
 import { useAuthState } from "react-firebase-hooks/auth";
 import { auth } from "../firebase.config";
 import { signOut } from "firebase/auth";
+import { HomePage } from "./components/HomePage"
+import { Navbar } from "./components/Navbar"
```

Comment:
Here you have imported `HomePage` and `Navbar` as a named import but you have exported it has a default export, so do this:
`import HomePage from "./components/HomePage";
`import Navbar from "./components/Navbar";